tests/functional/modules/test_zos_blockinfile_func.py

```diff
@@ -1517,102 +1518,41 @@ def test_special_characters_ds_insert_block(ansible_zos_module):
 def test_uss_encoding(ansible_zos_module, encoding):
     hosts = ansible_zos_module
     insert_data = "Insert this string"
-    params = {
```

Comment:
Wasn't this a change Isaac did to make the tests more _pythonic_? why is this changed?